R/Spectra.R

```diff
@@ -1565,6 +1565,11 @@ setMethod("containsMz", "Spectra", function(object, mz = numeric(),
                                             ppm = 20, which = c("any", "all"),
                                             BPPARAM = bpparam()) {
     cond_fun <- match.fun(match.arg(which))
+    if (!identical(order(mz), seq_along(mz))) {
+        message("Ordering mz values.")
```

Comment:
Do we need this message at all? E.g. in `MsCoreUtils::between` we silently reorder the `range` argument if it is unsorted. If we want to keep this message we could add a `verbose` or `quiet` argument (in `MSnbase` we have `Msnbase::isMsnbaseVerbose()`, do we need something similar in `Spectra`?).

Reply:
I don't mind. It's easy enough to silence with suppressMessage(). The reason I put the message in is a bit more complex (see #260).  

Let's drop it here.